src/engraving/compat/midi/compatmidirenderinternal.cpp

```diff
@@ -243,6 +244,23 @@ static void playNote(EventsHolder& events, const Note* note, PlayNoteParams para
     }
 
     events[params.channel].insert(std::pair<int, NPlayEvent>(std::max(0, params.onTime - params.offset), ev));
+    Accidental* acc = note->accidental();
+    if (acc) {
+        auto type = acc->accidentalType();
+        auto cents = Accidental::subtype2centOffset(type);
```

Comment:
Please do not use `auto` for simple types and when the type is known

Reply:
fixed